### ISSUE TITLE
ava/no-import-test-files does not detect helpers imported with extensions.

### ISSUE THREAD

**Issue by coreyfarrell** [avajs/eslint-plugin-ava#261]
The following code in a test file causes `ava/no-import-test-files` to raise an error
```js
import * as helpers from './_helpers.js';
```

The issue does not occur with `import * as helpers from './_helpers';`.  The project in question has the following custom rule configured to `xo` to enforce code that is closer to the way that node.js native ESM will work:
```js
"import/extensions": [
	"error",
	"always",
	{
		"ignorePackages": true
	}
]
```

On the other side it looks like this rule does not catch import's of test files when they do not specify an extension:
```js
import './some-test-file';
```

**Comment by novemberborn** [comment]
Could you share your `eslint-plugin-ava` version (as actually installed), your AVA version and the AVA configuration?

**Comment by coreyfarrell** [comment]
It appears that `xo@latest` is installing 5.1.1.  I just forced it to use eslint-plugin-ava@7.1.0 instead and it appears to have fixed the issue.  Sorry I didn't check this before posting.  I'm guessing this will just have to wait for xo@0.25.0 to drop node.js 6 and bump all dependencies.
